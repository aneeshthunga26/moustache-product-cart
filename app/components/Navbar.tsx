import MiniCartButton from "./MiniCartButton";

/**
 * Navbar that appears on top of the webpage which contains the Mini cart
 * @returns Navbar JSX Element
 */
export default function Navbar() {
  return (
    <nav
      className="navbar m-auto mt-4 flex-col justify-center items-end bg-gray-100 h-8 w-8/12 px-40"
      data-testid="navbar"
    >
      <div className="mr-10">
        <MiniCartButton />
      </div>
    </nav>
  );
}
