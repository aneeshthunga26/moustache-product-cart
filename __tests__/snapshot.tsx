import Navbar from "@/app/components/Navbar";
import { render } from "@testing-library/react";

it("renders navbar unchanged", () => {
  const { container } = render(<Navbar />);
  expect(container).toMatchSnapshot();
});
