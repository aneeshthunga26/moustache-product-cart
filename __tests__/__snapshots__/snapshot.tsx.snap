// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navbar unchanged 1`] = `
<div>
  <nav
    class="navbar m-auto mt-4 flex-col justify-center items-end bg-gray-100 h-8 w-8/12 px-40"
    data-testid="navbar"
  >
    <div
      class="mr-10"
    >
      <div
        class="dropdown-end dropdown foc"
      >
        <span
          class="text-gray-500 font-normal text-sm"
          tabindex="0"
        >
          My cart
        </span>
        <div
          class="card border-gray-600 dropdown-content mt-2 card-compact bg-white z-30 w-52"
          tabindex="0"
        >
          <div
            class="card-body"
          >
            Cart Items
          </div>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
